ui/src/App.tsx

```diff
@@ -15,13 +21,16 @@ import {
 import Refresh from '@mui/icons-material/Refresh';
 import { brown } from '@mui/material/colors';
 
+import gridButton from './components/GridButton';
```

Comment:
```suggestion
import GridButton from './components/GridButton';
```

Components are always Capitalized